scripts/_wait4_all.js

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env node
+
+const { execSync } = require('child_process')
+
+/**
+ * @file _wait4_all.js
+ * @description Waits for all docker-compose services to be running and healthy
+ */
+
+// Define the docker-compose containers you want to monitor here
+const expectedContainers = [
+  'account-lookup-service',
+  'central-ledger',
+  'kafka',
+  'ml-api-adapter',
+  'mysql',
+  'mysql_als',
+  'objstore',
+  'quoting-service',
+  'simulator',
+  'transaction-requests-service'
+]
+
+let retries = 15
+const waitTimeMs = 10000
+
+async function main () {
+  const waitingMap = {}
+  // serviceName => status, where status is unknown, healthy, unhealthy or starting
+  expectedContainers.forEach(serviceName => {
+    waitingMap[serviceName] = 'unknown'
+  })
+
+  try {
+    while (retries > 0) {
+      await updateServiceStatus(waitingMap)
```

Comment:
what about moving this block into a separate function, so the while-loop will be easy to understand?